Biasing/include/Biasing/TargetDarkBremFilter.h

```diff
@@ -117,8 +118,10 @@ class TargetDarkBremFilter : public simcore::UserAction {
    */
   inline bool isOutsideTargetRegion(const G4LogicalVolume* vol) const {
     if (!vol) return true;
+    static G4Region* TargetRegion =
```

Comment:
I am a bit skeptical about storing pointers in static memory. Is there a reason for not just having this defined as a member variable ?